biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/Activator.java

```diff
@@ -119,6 +122,9 @@ public boolean active() {
 	public void run() {
 
 		continuous = Boolean.valueOf(context.getProperty(TESTER_CONTINUOUS));
+		terminate = Optional.ofNullable(context.getProperty(TESTER_TERMINATE))
+			.map(Boolean::valueOf)
+			.orElse(true);
```

Comment:
The default value is documented to be `false` in `Syntax` and is initialized to `false` in the constructor.

Reply:
default should be true, sorry, fixing.

Reply:
this code doesn't need to change however.